code/modules/mob/living/simple_animal/hostile/megafauna/hierophant.dm

```diff
@@ -595,8 +596,6 @@ Difficulty: Hard
 		var/limb_to_hit = L.get_bodypart(pick("head", "chest", "r_arm", "l_arm", "r_leg", "l_leg"))
 		var/armor = L.run_armor_check(limb_to_hit, "melee", "Your armor absorbs [src]!", "Your armor blocks part of [src]!", 50, "Your armor was penetrated by [src]!")
 		L.apply_damage(damage, BURN, limb_to_hit, armor)
-		if(ismegafauna(L) || istype(L, /mob/living/simple_animal/hostile/asteroid))
-			L.adjustBruteLoss(damage)
```

Comment:
This change means it's as effective against megafauna as it was before, and against asteroid monsters buffing it to 20 damage would actually make it deal more damage than this change. I'd also argue that having it have more power as a "fighty weapon" is good, since it's generally more interesting to have traitors using weapons from their jobs rather than the same old uplink stuff.